.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": false,
+  "semi": true,
+  "trailingComma": "all"
+}
```

Comment:
This `prettier` config comes fairly close to matching the existing format in most of our files.